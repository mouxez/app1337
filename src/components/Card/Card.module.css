@import "../../globalStyles/variables.css";

.container {
    padding: 16px;
    border-radius: 4px;
    background-color: var(--primary-white);
    box-shadow: -10px 0px 13px -7px #000000, 10px 0px 13px -7px #000000, 5px 5px 15px 5px rgba(0, 0, 0, 0);
}