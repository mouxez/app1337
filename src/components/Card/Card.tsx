import { FC, ReactNode } from "react";
import styles from "./Card.module.css";

const { container } = styles;

type CardProps = {
  children: ReactNode;
  className?: string;
};

const Card: FC<CardProps> = ({ children, className = "" }) => {
  return <div className={`${container} ${className}`}>{children}</div>;
};

export default Card;
