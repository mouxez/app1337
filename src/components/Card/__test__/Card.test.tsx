import { render } from "@testing-library/react";
import Card from "../Card";

describe("components/Card", () => {
  it("renders correctly", async () => {
    const { asFragment } = render(
      <Card>
        <span>I am a child!</span>
      </Card>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
