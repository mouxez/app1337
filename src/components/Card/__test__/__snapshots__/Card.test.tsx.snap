// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Card renders correctly 1`] = `
<DocumentFragment>
  <div
    class="container undefined"
  >
    <span>
      I am a child!
    </span>
  </div>
</DocumentFragment>
`;
